src/test/java/org/jabref/model/groups/TexGroupTest.java

```diff
@@ -18,7 +17,7 @@
 
     @Test
     public void containsReturnsTrueForEntryInAux() throws Exception {
-        Path auxFile = Paths.get(AuxParserTest.class.getResource("paper.aux").toURI());
+        Path auxFile = Paths.get(TexGroupTest.class.getResource("paper.aux").toURI());
```

Comment:
Whats the reason for this change? The only "advantage" is that we now have a copy of the `paper.aux` file...

Reply:
I had the fear that maintainers of the AuxParserTest do not know that the TexGroupTest also relies on the aux file. What if another test is created? I would separate tests of different classes as much as possible even if files (`.bib`, `.aux`, ...) are duplicated.